www/includes/easyparliament/templates/html/mp/divisions.php

```diff
@@ -106,8 +106,10 @@
                                         ?>
                                         <?php foreach ($policy['divisions'] as $division) { ?>
                                             <li id="<?= $division['division_id'] ?>" class="<?= $division['strong'] || $show_all ? 'policy-vote--major' : 'policy-vote--minor' ?>">
+                                                <a name="<?= $division['division_id'] ?>"></a>
```

Comment:
That would also mean you could use `::target` for style changes, no need for a selected class.